pubspec.lock

```diff
@@ -1,20 +1,6 @@
 # Generated by pub
```

Comment:
need delete pubspec.loc, my mistake
and then need add pubspec.lock in .gitignore for necxt commit